extra_docs/quickstart.md

```diff
@@ -1,30 +1,29 @@
-# Quick Start Guide for the Javascript SDK
+# Quick Start Guide for Jexia Javascript SDK
 
-The SDK currently exposes the following features:
-  - on-demand access to data stored in DataSets (creating, reading, updating, deleting records)
-  - Authentication and authorization is handled automatically by the SDK, the user only needs to provide credentials once at SDK initialization.
+Jexia SDK currently exposes the following features:
+  - On-demand access to data stored in Datasets and Filesets (CRUD operations, file uploads)
+  - Authentication and authorization under the platform, you only need to provide your credentials.
 
-The SDK is currently focused on *consuming* data. *Managing* the data schema (creating Datasets, adding columns to Datasets, etc.) is out of scope for now. The data schema management is done through our web application at [app.jexia.com](app.jexia.com).
+Currently SDK is focused on *consuming* data. In order to manage data schema (creating Datasets, adding fields to them, etc.) is done through our web application at [app.jexia.com](https://app.jexia.com).
```

Comment:
Please, link with this- https://docs.jexia.com/getting-started/start-your-first-project/ instead app.jexia.com

Reply:
changed to: 
> Currently SDK is focused on *consuming* data. In order to manage data schema (creating Datasets, adding fields to them, etc.) you need to use our [web application](https://app.jexia.com). We suggest you read the guide [Start your first project](https://docs.jexia.com/getting-started/start-your-first-project/).

Reply:
Great! I like it